### PR TITLE
Add the date to posts

### PR DESCRIPTION
**Pull request by aileenrose**

### CODE REVIEW

**Comment by ryan-blunden** [comment]
Could we keep the formatting the same as the blog index page so it's all on one line?

**Comment by aileenrose** [comment]
> Could we keep the formatting the same as the blog index page so it's all on one line?

I'll make it the same.

**Comment by aileenrose** [comment]
@ryan-blunden done!

File: website/src/templates/blogPostTemplate.tsx [hunk 1]
```diff
@@ -34,18 +34,16 @@ export default class BlogPostTemplate extends React.Component<any, any> {
...
+    // Question: Should this be a Function Component?
     public Author(props: AuthorProps): Element {
         let element: any
 
         if (props.authorUrl) {
             element = (
-                <p>
-                    Written by <a href={props.authorUrl}>{props.author}</a>
-                </p>
+             <span>by <a href={props.authorUrl}>{props.author}</a></span>
```
Comment (ryan-blunden) [inline website/src/templates/blogPostTemplate.tsx]:
```suggestion
             <span>By <a href={props.authorUrl}>{props.author}</a></span>
```

File: website/src/templates/blogPostTemplate.tsx [hunk 2]
```diff
@@ -34,18 +34,16 @@ export default class BlogPostTemplate extends React.Component<any, any> {
...
             )
         } else {
-            element = <p>Written by {props.author}</p>
+            element = <span>by {props.author}</span>
```
Comment (ryan-blunden) [inline website/src/templates/blogPostTemplate.tsx]:
```suggestion
            element = <span>By {props.author}</span>
```

**Pull request merged**